src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -508,22 +511,27 @@ export default class QuestionnaireForm extends Component {
         // const priorAuthUrl = "http://localhost:9000/fhir/Claim/$submit";
         Http.open("POST", priorAuthUrl);
         Http.setRequestHeader("Content-Type", "application/fhir+json");
-        Http.send(JSON.stringify(priorAuthBundle));
+        Http.send(JSON.stringify(priorAuthBundle));          
+        var qForm = this;        
         Http.onreadystatechange = function() {
             if (this.readyState === XMLHttpRequest.DONE) {
                 var message = "";
-                if (this.status === 200) {
+                if (this.status === 200) {                   
                     var claimResponse = JSON.parse(this.responseText);
                     message = "Prior Authorization " + claimResponse.disposition + "\n";
-                    message += "Prior Authorization Number: " + claimResponse.preAuthRef;
+                    message += "Prior Authorization Number: " + claimResponse.preAuthRef;                 
                 } else {
                     message = "Prior Authorization Request Failed."
                 }
                 console.log(message);
                 alert(message);
-                console.log(this.responseText);
+                console.log(this.responseText);      
+                
+                // DME Orders                
```

Comment:
Should this just be moved within the check for status == 200 above? I think it might be a little bit cleaner instead of checking it again.